subprojects/launcher/src/integTest/groovy/org/gradle/launcher/continuous/SmokeContinuousIntegrationTest.groovy

```diff
@@ -300,9 +417,11 @@ class SmokeContinuousIntegrationTest extends AbstractContinuousIntegrationTest {
     def "failure to determine inputs cancels build and has a reasonable message after initial success"() {
         when:
         def bFlag = file("bFlag")
+        file("inputA").createFile()
```

Comment:
What breaks if we don't create this file? Why is this now necessary?

Reply:
If there are only missing files in the root project, then we are not watching the root project. So the only input here must not be missing.